README.md

```diff
@@ -1,18 +1,23 @@
-# @polkadot/apps
+# multisig wallet
 
-A Portal into the Polkadot and Substrate networks. Provides a view and interaction layer from a browser.
+## Introduction
 
-This can be accessed as a hosted application via https://polkadot.js.org/apps/ or you can access the IPFS hosted version via https://polkadot.js.org/apps/ipfs (via hash) or https://dotapps.io (via ipns) to explore any of the supported Polkadot and Substrate chains.
 
-If you run one or more IPFS node(s), pinning the UI (which only gets updated on releases) will make it faster for you and others. You can find details about that below in the IPFS chapter below.
+A Multisig (multiple signature) wallet requires multiple signers to approve transactions, offering enhanced security and collective management over funds. This is useful for organizations or groups looking to share control of a wallet among multiple parties.      
 
-**Important** If you are a chain developer and would like to add support for your chain to the UI, all the local configuration (API types, settings, logos) can be customized in [the apps-config package](packages/apps-config#README.md), complete with instructions of what goes where.
+This Comsig wallet allows users to securely create and manage wallets with multiple signers on Polkadot chains. Designed with enhanced security and collaboration in mind, users can easily set up wallets that require the approval of multiple parties for transactions and wallet management, ensuring transparency and protection for all stakeholders.
```

Comment:
since it's targeting several substrate-based network like bittensor, commune ai, and other chains, don't call it "comsig"